packages/web-components/src/appointment-bookings/client/components/time-cell.svelte

```diff
@@ -0,0 +1,24 @@
+<script>
+  import { css } from 'emotion'
```

Comment:
No need to include emotion in Svelte files, it's trivial just to add or remove the class without